src/pages/trainer/training/edit/components/toolbar/icons/italicIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+export const ItalicIcon = () => {
+  return (
+    <span className="glyphicon glyphicon-italic"/>
+  );
+};
```

Comment:
I think, @brauliodiez is saying something like:

**.../toolbar/icons/simpleIcon.tsx**
```javascript
import * as React from 'react';
 IProps interface {
   icon: string;
 }
 export const SimpleIcon = (props: IProps) => {
   return (
     <span className={props.icon}/>
   );
 };
```

**.../toolbar/icons/doubleIcon.tsx**
```javascript
import * as React from 'react';
 IProps interface {
   iconOne: string;
   iconTwo: string;
 }
 export const DoubleIcon = (props: IProps) => {
   return (
     <div>
      <span className={props.iconOne}/>
      <span className={props.iconTwo}/>
    </div>
   );
 };
```